material-ui/material-ui.scss

```diff
@@ -120,3 +121,120 @@ $swal2-close-button-hover-color: $swal2-close-button-color;
     }
   }
 }
+
+// Checkbox
+.swal2-checkbox {
+  input {
+    position: relative;
+    left: 30px;
+    width: 40px;
+    height: 40px;
+    margin: 0;
+    padding: 20px;
+    transform: scale(1);
```

Comment:
Do you mind explaining the goal of this `transform` here?

Reply:
I was playing with this to see if it would have an effect on the location of the hover shadow, but I realize it is unnecessary. I will remove it.